src/app/client/src/app/modules/groups/components/add-member/add-member.component.ts

```diff
@@ -112,7 +112,10 @@ export class AddMemberComponent implements OnInit, OnDestroy {
 
   verifyMember() {
     this.showLoader = true;
-      this.groupsService.getUserData((this.memberId).trim(), {token: this.captchaResponse})
+    const trimExtraSpace = this.memberId.split(' ');
```

Comment:
why are we splitting the **memberId** ?

Reply:
no actually i split only the unwanted space.

Reply:
it won't work. for example: 
**memberId = "kamalee _31466"** , 
here i have given extra space by mistake, now what split will do is: 
**["kamalee", "_31466"]**

now if u take 0th index, then it will be wrong, isn't it ?